### ISSUE TITLE
Git Issues Gantt Chart Enhancements

### ISSUE THREAD

**Issue by vojtechsedlak** [MozillaFoundation/Advocacy#324]
Detailed description of work: 
Let's keep track of changes and improvements needed for the Gantt chart visualization (see [here](https://mozilla-advocacy-gantt-pr-1.herokuapp.com/)). There are some issues at the moment, which I am looking into.

_Start date: September 6, 2016_ 
_Due date: September 16, 2016_ 

CC: @valianttry @damiankay @cadecairos

**Comment by damiankay** [comment]
@valianttry had requested the ability to sort by label. 

CC @vojtechmozilla

**Comment by damiankay** [comment]
I think it's worth re-inserting the example dates that you had previously. I only meant to remove the parentheses, not the enclosed content. 

@vojtechmozilla

**Comment by vojtechsedlak** [comment]
@valianttry I've added filtering by labels. Please let me know if you run into any issues. Thanks!

**Comment by damiankay** [comment]
Hey @valianttry & @cadecairos 

I've tried the staging and prod links for the Gantt chart, and neither works. Can you investigate please? 

New Gantt chart: https://mozilla-advocacy-gantt-pr-1.herokuapp.com/
We’ll ship it to https://advocacy-gantt.mofoprod.net/

**Comment by vojtechsedlak** [comment]
I am happy to look into this today @damiankay, I am working out of the SF office.

**Comment by vojtechsedlak** [comment]
Accidentally closed this issue. @damiankay the gantt chart should be working now, I have added error reporting for git tickets that are improperly formatted (which was the cause of the gantt chart breaking).

**Comment by damiankay** [comment]
Great - thanks @vojtechmzl

**Comment by damiankay** [comment]
Will we be moving to the production URL, or not until you're satisfied that there won't be more changes?

**Comment by valianttry** [comment]
When you select either "all, P1, or P2" the button doesn't maintain a new state after it is selected so you can''t discern what's selected. I sorted by "Start Date" and selected "P1" and this is what the buttons look like:

![github_issues_gantt](https://cloud.githubusercontent.com/assets/9830371/18606065/0ae99554-7c72-11e6-9524-9dc7ddd91295.png)

You should be able to see what you selected until a new selection is made.

**Comment by valianttry** [comment]
@vojtechmzl your label color is transparent. It's a color in the actual chart, but the key has a blank color for you (Firefox on  Mac):

![screen_shot_2016-09-17_at_12_53_35_am](https://cloud.githubusercontent.com/assets/9830371/18606046/65251d8c-7c71-11e6-9a89-8940b8a32fe4.png)

**Comment by valianttry** [comment]
The scroll bar is doing whacky things. It's not usable (I now have tickets in Nov-Dec but can't scroll right past Nov 20. Firefox on  Mac.

![github_issues_gantt_and_desktop](https://cloud.githubusercontent.com/assets/9830371/18606054/c2b5f5e8-7c71-11e6-8e94-a71a156b4cbf.png)

**Comment by vojtechsedlak** [comment]
I fixed the issues mentioned above @valianttry. Please let me know if there are any other issues @damiankay. Otherwise we can move this to production, with the help of @cadecairos, and close this issue. Thanks!

**Comment by cadecairos** [comment]
I'll add this to my list of things so I don't lose it - let me know when it's ready to go!

**Comment by vojtechsedlak** [comment]
Closing this ticket now that production has been updated. If you uncover new issues or have new feature requests @valianttry @damiankay, let me know or open a new ticket. Thanks!
